WordPress/src/main/java/org/wordpress/android/ui/stockmedia/StockMediaRetainedFragment.java

```diff
@@ -0,0 +1,43 @@
+package org.wordpress.android.ui.stockmedia;
+
+import android.app.Fragment;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+
+import org.wordpress.android.fluxc.model.StockMediaModel;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class StockMediaRetainedFragment extends Fragment {
```

Comment:
This is just a personal note:

I am really surprised to see that this fragment is only used as a glorious Bundle :) I'd have expected it to be source of truth for the data, which would have made it awful close to a ViewModel and would require similar patterns to it which was a concern for me. I am glad to see this solution while we are documenting the `ViewModel` pattern as this would have been a great candidate for it as you already tried it out.